packages/token-list/src/index.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * External dependencies
+ */
+import { uniq, compact, without } from 'lodash-es';
```

Comment:
Do you think we should just use `lodash` for now and update to `lodash-es` consistently across packages in a separate PR?

Reply:
Yes, that seems reasonable.